### PR TITLE
LPD-30611

### PR DESCRIPTION
**Pull request by vngoliveira**
[LPD-30611](https://liferay.atlassian.net/browse/LPD-30611)

### CODE REVIEW

**Comment by vngoliveira** [comment]
ci:test:sf

File: modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/resource/v2_0/DataDefinitionResourceImpl.java [hunk 1]
```diff
@@ -464,67 +464,26 @@ public DataDefinition putDataDefinition(
...
-			dataDefinition, dataDefinitionId, ddmForm);
+		dataDefinition = _putDataDefinition(
+			dataDefinitionId, dataDefinition, ddmStructure);
 
 		for (long classPK :
 				_deDataDefinitionFieldLinkLocalService.getClassPKS(
 					_portal.getClassNameId(DDMStructure.class),
 					dataDefinitionId)) {
 
+			DDMStructure nestedDDMStructure =
```
Comment (marcelabc) [inline modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/resource/v2_0/DataDefinitionResourceImpl.java:475]: Can you rename it to existingDDMStructure?

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

**Comment by marcelabc** [comment]
I will run the CI to antecipate results.

**Comment by marcelabc** [comment]
ci:test:data-engine

**Comment by vngoliveira** [comment]
ci:test:sf

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:data-engine

**Comment by vngoliveira** [comment]
Hey @marcelabc, I ran the tests locally, and they passed. I will rerun the data engine here to recheck the results.

**Comment by vngoliveira** [comment]
ci:test:data-engine

**Comment by marcelabc** [comment]
ci:report:36027667

**Comment by marcelabc** [comment]
ci:forward

**Pull request closed without merging**